### PR TITLE
Output plugins in feed

### PR DESCRIPTION
**Pull request by pathawks**
This site has no posts; the feed is empty and unnecessary.

### CODE REVIEW

**Comment by parkr** [comment]
It'd be cool to be able to subscribe to new plugins as they're added. Maybe something like that could work? Looks like we could customize this to use the right collection.

**Comment by pathawks** [comment]
I have replaced the feed with a modified version of [the template used by **jekyll-feed**](https://github.com/jekyll/jekyll-feed/blob/544042151bba72352562045a28f02c2a16729c7c/lib/feed.xml).

It will output all plugins as feed entries. We do not know when each plugin was added, so we cannot sort them or limit to a certain number of entries. We can only output all of them and leave it up to feed readers to do the right thing.

~~I am using Jekyll 3.3 filters, so it is probably best to wait until GitHub Pages supports 3.3 before merging this.~~

**Comment by pathawks** [comment]
I guess this is another instance where collections support in **jekyll-feed** would be cool, and also an example of the complexity that comes with trying to create a feed of items with no date attached.

jekyll/jekyll-feed#124

File: feed.xml
Comment (parkr) [inline feed.xml:27]: Why is this commented out?

**Review by parkr — APPROVED** [review]

File: feed.xml
Comment (pathawks) [reply feed.xml:27]: This is a Liquid trick. All the `assign` still happens, but it does not output all the newlines and tabs in the output.

Comment (ashmaroli) [reply feed.xml:27]: The [documented](https://shopify.github.io/liquid/basics/whitespace/) way to have this done is to use `hyphens '-'`

```
{%- assign post_author = post.author | default: post.authors[0] | default: site.author -%}
```

This makes it future-proof if they do have liquid ignore everything within the 
`{% comment %} {% endcomment %}` tags

Comment (pathawks) [inline feed.xml]: Wait, WHAT???

Comment (pathawks) [reply feed.xml:27]: Does not seem to work in our version of Liquid:

``` text
Liquid Exception: Liquid syntax error (line 18): Tag '{%- assign post_author = post.author | default: post.authors[0] | default: site.author -%}' was not properly terminated with regexp: /\%\}/ in feed.xml
```

**Edit:** Added in Shopify/liquid#773 and will come with Liquid 4.

Comment (ashmaroli) [reply feed.xml:27]: you missed the hyphen at the termination `%}`

**Pull request merged**
